release-kong.sh

```diff
@@ -1,230 +1,125 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+
 CWD=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+BUILD_DIR="$CWD/output"
+
+
+PULP_USERNAME="$PULP_USR"
+PULP_PASSWORD="$PULP_PSW"
+PULP_DOCKER_IMAGE="kong/release-script"
```

Comment:
I kind of wish we had a `kong/release-scripts:1` so this can be pinned but only to the major. I suppose it's fine it's an image we own so we should know to not break backwards compatibility